provers/tests/src/provers/btc.rs

```diff
@@ -60,18 +53,21 @@ mod test {
     #[test]
     #[cfg(not(any(feature = "risc0", feature = "sp1")))]
     fn test_native() {
-        test_proof(hosts::native::btc_blockspace());
+        use crate::provers::TEST_NATIVE_GENERATORS;
+        test_proof(TEST_NATIVE_GENERATORS.btc_blockspace());
     }
 
     #[test]
     #[cfg(feature = "risc0")]
     fn test_risc0() {
-        test_proof(hosts::risc0::btc_blockspace());
+        use crate::provers::TEST_RISC0_GENERATORS;
+        test_proof(TEST_RISC0_GENERATORS.btc_blockspace());
     }
 
     #[test]
     #[cfg(feature = "sp1")]
     fn test_sp1() {
-        test_proof(hosts::sp1::btc_blockspace());
+        use crate::provers::TEST_SP1_GENERATORS;
```

Comment:
all these `use::provers::TEST_FOO_GENERATORS;` can be added to the top of the file? I don't like imports flying under the radar.

Reply:
Ok I think it's fine for us to break convention here.